homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -154,13 +165,15 @@
 ATTR_HARDWARE_VERSION = "hardware_version"
 
 # Air Humidifier CA
-ATTR_MOTOR_SPEED = "motor_speed"
+# ATTR_MOTOR_SPEED = "motor_speed"
```

Comment:
Why change this?

Reply:
Duplicates ATTR_MOTOR_SPEED on line 139. Didn't drop this entirely just to keep the notion of this attribute in this "Air Humidifier CA" section